sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -57,16 +57,23 @@ class GpuProjectExecMeta(
     val gpuExprs = childExprs.map(_.convertToGpu().asInstanceOf[NamedExpression]).toList
     val gpuChild = childPlans.head.convertIfNeeded()
     if (conf.isProjectAstEnabled) {
-      if (childExprs.forall(_.canThisBeAst)) {
+      // cuDF requires result columns are fixed width
+      val resultTypes = gpuExprs.map(_.dataType)
+      val isFixedWidth = resultTypes.forall(GpuBatchUtils.isFixedWidth)
```

Comment:
Nit: We are checking all columns to be fixed width so use `all*`.
```suggestion
      // cuDF requires result columns are fixed width
      val resultTypes = gpuExprs.map(_.dataType)
      val allFixedWidth = resultTypes.forall(GpuBatchUtils.isFixedWidth)
```

Reply:
Done